README.md

```diff
@@ -1,3 +1,35 @@
 # Starknet in Rust
 
 `starknet_in_rust` is a implentation of [Starknet](https://github.com/starkware-libs/cairo-lang/tree/master/src/starkware/starknet) in Rust.
```

Comment:
little typo, `is an implementation` instead of `is a implementation`

Reply:
IIRC we should use `an` since i is a vowel.

Reply:
yes exactly, but `a` is being used, sorry if didn't explain myself correctly.